baseapp_core/hashids/strategies/public_id/drf_resolver.py

```
@@ -0,0 +1,9 @@
+from baseapp_core.hashids.strategies.interfaces.id_resolver import IdResolverStrategy
+
+
+class PublicIdDRFResolverStrategy:
+    def __init__(self, id_resolver: IdResolverStrategy):
+        self.id_resolver = id_resolver
+
+    def resolve_public_id_to_pk(self, id, expected_model=None):
```

Ruff (0.14.10)

Unused method argument: `expected_model`

(ARG002)